app/src/main/res/xml/network_security_config.xml

```diff
@@ -5,5 +5,7 @@
         <domain includeSubdomains="true">134.122.59.107</domain>
         <!--        This is for RegTest faucet, to claim some starter money -->
         <domain includeSubdomains="true">131.180.27.224</domain>
+        <!--        IP address of the host machine when ran from the Android emulator -->
+        <domain includeSubdomains="true">10.0.2.2</domain>
```

Comment:
I don't think this should be merged into production code.

Reply:
I've commented it out and left a note in README that in case the local development in the emulator is done, the entry has to be uncommented.